www/test/functional/report.spec.js

```diff
@@ -15,33 +15,34 @@ describe('Report address form', function() {
     report = new Report();
     confirmation = new Confirmation();
     placeDetails = new PlaceDetails();
-
   });
 
   it('should not send to confirmation page without address', function() {
     browser.get('/#/report');
+    var date = moment('2015-02-25T02:30:00').format('MM/DD/YYYY, HH:mm');
+    report
+      .fillDateWithFormat(date)
+      .fillRisk('Roubo');
```

Comment:
This is the result of returning `this` from that methods I changed. We can now call the methods like this. I think it turns easier to read, looks like a document.